core/rfb.js

```diff
@@ -67,6 +75,12 @@ export default class RFB extends EventTargetMixin {
         this._target = target;
         this._url = url;
 
+        this._gestureLastTapTime = null;
+        this._gestureFirstDoubleTapEvent = null;
```

Comment:
That's a very long name, I'd say `_gestureFirstDoubleTap` is enough.

Reply:
Then you lose the context of what type of object is stored in this variable. Besides, JavaScript loves long names. :)

Reply:
Fine, I had already renamed it in 6d57ee9, but it's now fixed in e59e62b